examples/soroban/nft/src/nft.rs

```diff
@@ -15,7 +15,7 @@ pub struct MyNonFungibleToken {
     admin: Address,
     name: Bytes,
     total_count: u32,
-    owners_to_nft_ids: Map<Address, u32>,
+    owners_to_nft_ids: Map<Address, Vec<u32>>,
```

Comment:
I realized that I made a mistake with the `owners_to_nfts_ids` field and was overwriting the owner's nft id if they had more than one. 🤦‍♀️ 

I added this field so that it would be easier to get all of a user's nfts... but I'm not sure it is worth adding it in here. What do you think @willemneal ?

Reply:
Makes sense here. You could use a` Map<u32, ()>`, turning it into a Set. So adding and removing will be easier and you can use `.keys()` to get the `Vec<u32>`

Reply:
Good call! Just pushed up the refactor